### DISCUSSION TITLE
unable to use -web.listen.address flag

### DISCUSSION THREAD

**Discussion by tune007** [burningalchemist/sql_exporter#519] [Q&A]
**Describe the bug**
I am getting an error when trying to use the web.listen.address flag
flag provided but not defined: -web.listen.address

**To Reproduce**
trying to run the flag with or without quotes in different ways like this gives me error
sql_exporter.exe --web.listen.address "servername.domain.com:9397"

**Expected behavior**
use specified target for web.listen-address

**Additional context**
please provide correct format for supplying this flag if I am doing it incorrect

**Comment by burningalchemist** [comment]
Hi @tune007, as per help (`./sql_exporter --help`), it should be `web.listen-address` (mind the second dash character). Give it a try?

**Comment by tune007** [comment]
thanks getting requested address is not valid in context now, any suggestions please ?

**Comment by tune007** [comment]
trying to setup a remote monitoring for an RDS instance

**Comment by burningalchemist** [comment]
Could you please post here a complete error message? It might be a different (connectivity) issue.
